packages/client/src/containers/Login.container/Login.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import SignupForm from './SignupForm';
+import './Login.css';
+
+export const Login = () => {
```

Comment:
Great, fyi there probably will be some merge conflicts with my pull - I did changes with authentication and created placeholder signup/login pages for firebase. So, please leave logic from my pages.